pybossa/jobs.py

```diff
@@ -622,21 +622,99 @@ def send_mail(message_dict, mail_all=False):
             mail.send(message)
 
 
-def delete_bulk_tasks(data):
-    """Delete tasks in bulk from project."""
+def count_rows_to_delete(project_id):
     from sqlalchemy.sql import text
     from pybossa.core import db
-    import pybossa.cache.projects as cached_projects
-    from pybossa.cache.task_browse_helpers import get_task_filters
 
-    project_id = data['project_id']
-    project_name = data['project_name']
-    curr_user = data['curr_user']
-    coowners = data['coowners']
-    current_user_fullname = data['current_user_fullname']
-    force_reset = data['force_reset']
-    url = data['url']
-    params = {}
+    sql = text('''
```

Comment:
Optional, the four select queries could be generalized into a helper method for code reuse:

```python
def get_count(table_name, project_id):
    sql = text(f'''
        SELECT COUNT(*) FROM {table_name} WHERE
        project_id=:project_id;
        ''')
    params = dict(project_id=project_id)
    return db.session.execute(sql, params).scalar()

def count_rows_to_delete(project_id):
    total_tasks = get_count('task', project_id)
    total_taskrun = get_count('task_run', project_id)
    total_result = get_count('result', project_id)
    total_counter = get_count('counter', project_id)

    return total_tasks, total_taskrun, total_result, total_counter
```